### PR TITLE
App launch perf fixes

### PR DESCRIPTION
**Pull request by jaredhms**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1534)

### CODE REVIEW

**Comment by yiyang-msft** [comment]
where _CreateWoosh gets disabled?

File: Frameworks/UIKit/UIView.mm [hunk 1]
```diff
@@ -504,7 +504,14 @@ - (UITouchPhase)_processPointerEvent:(WUXIPointerRoutedEventArgs*)pointerEventAr
         touchEventName = @selector(touchesCancelled:withEvent:);
     }
 
-    if (touchPhase != UITouchPhaseBegan && ![touchPoint.touch->_view->priv->currentTouches containsObject:touchPoint.touch]) {
+    if (!touchPoint.touch->_view) {
+        // Ignore if the pointer isn't captured
```
Comment (yiyang-msft) [inline Frameworks/UIKit/UIView.mm]: >isn't captured [](start = 33, length = 14)

when it is nil, is it because the touch get cancelled?

**Comment by yiyang-msft** [comment]
and may need a work item on that.

---
In reply to: [266920587](https://github.com/Microsoft/WinObjC/pull/1534#issuecomment-266920587) [](ancestors = 266920587)

**Comment by yiyang-msft** [comment]
:shipit:

**Comment by jaredhms** [comment]
it's disabled in layeranimation.mm

---
In reply to: [266921436](https://github.com/Microsoft/WinObjC/pull/1534#issuecomment-266921436) [](ancestors = 266921436,266920587)

**Pull request merged**